docs_app/tools/decision-tree-generator/src/tree.yml

```diff
@@ -8,9 +8,6 @@
         - label: to be a value calculated through a formula
           children:
             - label: map
-    - label: I want to pick a property off each emitted value
```

Comment:
I would love to leave this scenario and instead of `pluck`, I'd rather point to `map` as it's completely valid scenario that a user could search for.